h/views.py

```diff
@@ -133,5 +189,6 @@ def includeme(config):
     config.add_route('help', '/docs/help')
     config.add_route('onboarding', '/welcome')
     config.add_route('stream', '/stream')
+    config.add_route('atom_stream', '/stream.atom')
```

Comment:
Also means they'll probably show up in order in the Pyramid routes introspection (such as with the `proutes` command).